adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -743,6 +747,90 @@ class ADAMContext(@transient val sc: SparkContext) extends Serializable with Log
   }
 
   /**
+   * This method gets the sort and partition map metadata from the header of the file
```

Comment:
`This method gets the sort and partition map metadata...` &rarr; `Extracts the sort and partition map metadata...`